src/list-action-group/list-action-group.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <button type="button" class="editor-btn-delete" (click)="onDelete.emit()">
```

Comment:
Its the fist time I see you doing this pattern. What is the reason for using `emit` instead of just calling the function?